data_steward/tools/load_archived_submission.py

```diff
@@ -0,0 +1,172 @@
+
+    :param client: Bigquery Client object
+    :param hpo_id: Identifies the HPO site
+    :return: bucket name for the HPO site as a string
+    :raises GoogleCloudError/TimeoutError
+    """
+    bucket_name_query = BUCKET_NAME_QUERY.render(
+        project=client.project,
+        dataset=bq_consts.LOOKUP_TABLES_DATASET_ID,
+        bucket_names_table=bq_consts.HPO_ID_BUCKET_NAME_TABLE_ID,
+        hpo_id=hpo_id.upper())
+
+    try:
+        bucket_names_result = client.query(bucket_name_query).result()
+        bucket_names = bucket_names_result.to_dataframe(
+        )['bucket_name'].to_list()
+        if len(bucket_names) > 1:
+            LOGGER.warning(
+                f'Found more than one bucket name for site {hpo_id}: {bucket_names}'
+            )
+        bucket_name = bucket_names[0]
+    except (GoogleCloudError, TimeoutError) as e:
+        LOGGER.error(f'Job failed with error {str(e)}')
+        raise e
+    return bucket_name
+
+
+def _filename_to_table_name(filename: str) -> str:
+    """
+    Converts a file path/URI to a table name
+
+    :param filename: filename or URI as /path/to/file.csv
+    :return: table_name as string, e.g. 'file' for /path/to/file.csv
+    """
+    return filename.split('/')[-1].replace('.csv', '').lower()
+
+
+def load_folder(dst_dataset: str, bq_client: BQClient, bucket_name: str,
+                prefix: str, gcs_client: GCSClient,
+                hpo_id: str) -> List[LoadJob]:
+    """
+    Stage files from a bucket to a dataset
+
+    :param dst_dataset: Identifies the destination dataset
+    :param bq_client: a BigQuery client object
+    :param bucket_name: the bucket in GCS containing the archive files
+    :param prefix: prefix of the filepath URI
+    :param gcs_client: a Cloud Storage client object
+    :param hpo_id: Identifies the HPO site
+    :return: list of completed load jobs
+    """
+    blobs = list(gcs_client.list_blobs(bucket_name, prefix=prefix))
+
+    load_jobs = []
+    for blob in blobs:
+        table_name = _filename_to_table_name(blob.name)
+        if table_name not in AOU_REQUIRED:
+            LOGGER.debug(f'Skipping file for {table_name}')
+            continue
+        schema = get_table_schema(table_name)
+        hpo_table_name = f'{hpo_id}_{table_name}'
+        fq_hpo_table = f'{bq_client.project}.{dst_dataset}.{hpo_table_name}'
+        destination = Table(fq_hpo_table, schema=schema)
+        destination = bq_client.create_table(destination)
+        job_config = LoadJobConfig()
+        job_config.schema = schema
+        job_config.skip_leading_rows = 1
+        job_config.source_format = 'CSV'
+        source_uri = f'gs://{bucket_name}/{blob.name}'
+        load_job = bq_client.load_table_from_uri(source_uri,
+                                                 destination,
+                                                 job_config=job_config,
+                                                 job_id_prefix=__name__)
```

Comment:
I like this, but I think when run as a stand alone script this will prefix the job with `__main__`.  You can view some other samples with this quick script.
```python
import os

print(f'name -> \t{__name__}')

print(f'file ->\t {__file__}')

#print(f'qualname -> \t {__qualname__}')

#print(f'module ->\t{__module__}')

def in_func():
    print(f'----------------------------------------------------')
    print(f'name -> \t{__name__}')

    print(f'file ->\t {__file__}')
    print(f'file ->\t {os.path.basename(__file__)[0:-3]}')
    print(f'file ->\t {__file__.split("/")[-1][0:-3]}')

    # These are intended to be called on objects and not as stand alone built-ins. 
#    print(f'qualname -> \t {__qualname__}')

#    print(f'module ->\t {__module__}')                                                                                                                                                                                                   


in_func()
```

Reply:
Thanks, I was thinking the main script which is calling the function should be highlighted. This would result in the same job prefix if the caller was itself or a different script. It is definitely more informative than '__main__', so I can use `__file__` and extract the filename for the prefix.

Reply:
Actually, I tested this a bit more and it looks like `__name__` just returns the imported string from the caller. So it would serve the same purpose anyway. I'll just use `__file__`.